src/api/handlers/webhook.ts

```diff
@@ -370,8 +368,7 @@ export async function upsertSlashCommands(
   guildID?: string,
 ) {
   const data = options.map((option) => {
-    // Use ... for content length due to unicode characters and js .length handling
-    if ([...option.name].length < 2 || [...option.name].length > 32) {
+    if (/^[\w-]{1,32}$/.test(option.name)) {
```

Comment:
We need to check if the name does not match the regex.
```suggestion
    if (!/^[\w-]{1,32}$/.test(option.name)) {
```